codalab/rest/interpret.py

```diff
@@ -158,6 +158,32 @@ def fetch_interpreted_worksheet(uuid):
     - resolve_interpreted_items: get more information about a worksheet.
     In the future, for large worksheets, might want to break this up so
     that we can render something basic.
+    Return: 
```

Comment:
Looking for suggestions on the documentation, the worksheet_info contains a lot of complexity so we really need a accurate and helpful documentation for later development

Reply:
Totally agree - we should probably make a Python object out of this (e.g., using `dataclass`)

Reply:
Probably should give broader context (either here or refer people to another place).
- source (raw worksheet_items in the DB, rendered as lines) [give example]
- blocks (semantic units that are rendered).

I wouldn't call it "block items" - I think items should always refer to raw items (because that's in the DB, which is annoying to change); just call them blocks everywhere.